nestkernel/recording_device.cpp

```diff
@@ -581,6 +581,25 @@ nest::RecordingDevice::calibrate()
 }
 
 void
+nest::RecordingDevice::post_run_cleanup()
+{
+  if ( B_.fs_.is_open() )
+  {
+    if ( P_.flush_after_simulate_ )
+      B_.fs_.flush();
```

Comment:
2) Yes. And also in hundreds of other places in this file. Can't I just disable the checker until #691 is merged? I don't really see a reason for fixing this all over again...